modules/modules.go

```diff
@@ -18,7 +14,7 @@ func init() {
 	if build.Release == "dev" {
 		SafeMutexDelay = 3 * time.Second
 	} else if build.Release == "standard" {
-		SafeMutexDelay = 8 * time.Second
+		SafeMutexDelay = 60 * time.Second
```

Comment:
The lock will be held until all modules have processed the consensus change, because ReceiveConsensusSetUpdate is now a blocking function. This will slow things down a lot.

I also don't want false positives whatsoever in our production code, but don't feel like the codebase is ready to be shipped without the safelocks.